OpenSim/Simulation/Model/Model.cpp

```diff
@@ -661,9 +661,15 @@ void Model::createMultibodyTree()
 
     auto bodies = getComponentList<Body>();
     for (auto& body : bodies) {
-        _multibodyTree.addBody( body.getAbsolutePathName(),
-                                body.getMass(), false,
-                                const_cast<Body*>(&body) );
+        try {
+            _multibodyTree.addBody( body.getAbsolutePathName(),
+                                    body.getMass(), false,
+                                    const_cast<Body*>(&body) );
+        } catch (const std::runtime_error& e) {
+            cout << "Duplicate Body names are not permitted. Please update "
+                 << "your model's .osim file accordingly." << std::endl;
```

Comment:
> I thought that would be caught elsewhere. 

Ah...that could be true; I'm not sure.


> The request in #1768 was to "add to the message…".

The cout could be just fine. In other places (e.g., [here](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Common/Component.cpp#L233)), we throw a new exception like so:

```
throw Exception("Can't have duplicate names (details: " + x.what() + ").");
```